cms/static/sass/views/_group-configuration.scss

```diff
@@ -59,12 +59,12 @@
           @extend %cont-truncated;
           @extend %t-title5;
           @extend %t-strong;
-          margin-right: ($baseline*14);
+          @include margin-right($baseline*14);
           color: $black;
 
           .toggle {
+            @include padding-left($baseline);
```

Comment:
I think we should keep it. The module has few corner cases which shows and hides some bugs.

I'll have to keep this PR focused on the RTL issues. I could, however, start another PR to address the non-RTL issues in the group configuration page.